docs/how-to-run-multiple-minio-server-instances-on-single-machine.md

```diff
@@ -18,7 +18,7 @@ Minio server is running on port 9002, 9003, 9004 with associated data directory
 **Minio server on port ``9002``**
 
 ```sh
-$ ./minio -C /home/minio/minio1/.minio server --address 192.168.1.11:9002 minio1/data1/
+$ ./minio -C ~/minio1/.m1config server --address 192.168.1.11:9002 ~/minio1/data1/
```

Comment:
you have to do either
`$ ./minio -C ~/.m1config server --address 192.168.1.11:9002 ~/data1/`
or
`$ ./minio -C ~/minio1/.config server --address 192.168.1.11:9002 ~/minio1/data/` (I prefer this as minio servers will be segregated clearly by directories)

`$ ./minio -C ~/minio1/.m1config server --address 192.168.1.11:9002 ~/minio1/data1/` is not OK because you already have the directory by name `minio1` and again you are using `.m1config` and `data1` inside it which is not needed, you can just use `.config` and `data`